test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -42,7 +42,7 @@
 
     It "should only load the specified version" {
         Import-Module TestModule -RequiredVersion 1.1
-        (Get-Module TestModule).Version | Should Be "1.1"
+        (Get-Module TestModule).Version | Should BeIn "1.1"
```

Comment:
Isn't `BeIn` only used for validating if the left side is contained in the collection on the right side?  Don't understand why this isn't just `Be`?

Reply:
as it turns out, there are multiple loaded modules which are named TestModule, thus a comparison with "Be" fails because it's a collection. I didn't want to rearchitect the tests.

Reply:
Got it. Closed.